localization/english/wc_uther_events_l_english.yml

```diff
@@ -3,4 +3,28 @@
  uther_events.001.a:0 "No, I cannot. I shall discard my hammer, forever."
  uther_events.001.b:0 "Damn it, I will put you down, Arthas."
- uther_events.001.c:0 "Lordaeron must be protected at any cost."
\ No newline at end of file
+ uther_events.001.c:0 "Lordaeron must be protected at any cost."
+ uther_tooltip:0 "You will try and convince Arthas otherwise, or abandon the investigation."
+ uther_win_tooltip:0 "You convince him successfully."
+ uther_lose_tooltip:0 "You fail to convince him."
+ uther_stay_tooltip:0 "You will stay in the investigation party."
+ uther_events.002.t:0 "The Culling"
+ uther_events.002.d:0 "I nearly shudder when I hear the words come out of Arthas' mouth? #italic Purge?#! Who are we to sacrifice the innocents of Stratholme in pursuit of eradicating this plague?\n\nSurely this does not coincide with what I have been teaching him, and to kill is not a task that should be taken so lightly. Surely, there must be another way to save Stratholme. I must contemplate what the best course of action is from here, and quickly."
+ uther_events.002.a:0 "I cannot watch you do this, Arthas!"
+ uther_events.002.b:0 "Maybe I can convince him otherwise..."
+ uther_events.003.t:0 "The Fate of Stratholme"
+ uther_events.003.d:0 "Fortunately, I have succeeded in convincing Arthas there is another way to proceed with Strathlome. But now, the real battle begins. We must divise another way to save the city, as it's citizens are in dire need of our help.\n\nWe should..."
+ uther_events.003.a:0 "...Quarantine the city instead."
+ uther_events.003.b:0 "...Evacuate everyone."
+ uther_events.003.c:0 "...do whatever Arthas wants."
+ uther_events.035.d:0 "Rejoice!\n\nArthas has decided on another plan. We will be trying something less... #italic extreme#!.\n\nBecause of [uther.GetName],"
+ uther_events.035.d_q:0 " we will be attempting the quarantine the city instead. This will be a more gentle approach to the situation, albeit we will have quite the problem on our hand if it fails."
+ uther_events.035.d_e:0 " the city will be evacuated. We must be careful to identify sick citizens, so they do not spread the plague. This method will be risky, but it is better than them being dead..."
+ uther_events.035.a:0 "Thank the light! He was brought to his senses."
+ uther_events.035.b:0 "Why did he even consider that?"
+ uther_events.004.d_betrayal:0 "To my surprise, Arthas has gone behind my back, and decided to purge Strathlome once again! I can't believe he's done this.\n\n"
+ uther_events.004.d:0 "My plan to convince him has failed, and now others must suffer for my decision. I do not know if I can continue to mentor Arthas under this condition, he has gone mad.\n\nToday marks an important day for my dear Kingdom of Lordaeron. One that must not be forgotten, I forbid it. Innocents will die at Arthas' hand, and I cannot stop him. May they forgive me."
```

Comment:
```suggestion
 uther_events.004.d:0 "[GetGlobalVariable('arthas_character').Char.GetFirstName] has gone mad! My attempts to convince [GetGlobalVariable('arthas_character').Char.GetHerHim] have failed, and now others must suffer for my lacking. I do not believe I can continue to mentor [GetGlobalVariable('arthas_character').Char.GetFirstName]  in this condition.\n\nThis day will be ever remembered as a grave historic trauma for my cherished Kingdom of Lordaeron. Innocents will die at [GetGlobalVariable('arthas_character').Char.GetFirstNamePossessive] hand, and I cannot stop him. May they and the $HighGodNameAlternate$ forgive me."
```

Reply:
feel free to take any or all of the suggestions here, i would be happy so long as you localize the names